internal/db/advisory_locks_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2025 Canonical.
+
+package db_test
+
+import (
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/frankban/quicktest/qtsuite"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/testutils/jimmtest"
+)
+
+type advisoryLocksSuite struct{}
```

Comment:
Now that you are not doing anything custom, you could switch this test to use the `dbSuite` like many of the other tests in the `db` package.

Reply:
Agreed! Updated